src/Umbraco.Web.Common/Filters/ValidateUmbracoFormRouteStringAttribute.cs

```diff
@@ -42,7 +42,9 @@ public void OnAuthorization(AuthorizationFilterContext context)
             {
                 if (context == null) throw new ArgumentNullException(nameof(context));
 
-                var ufprt = context.HttpContext.Request.Form["ufprt"];
+                var ufprt = context.HttpContext.Request.HasFormContentType
+                    ? context.HttpContext.Request.Form["ufprt"]
+                    : context.HttpContext.Request.Query["ufprt"];
```

Comment:
Good thinking. It would defo make sense that when resolving this value it uses the same code / extension so that you don’t get these inconsistencies of implementation 👍🏻

Reply:
@mattbrailsford you up for picking this up?

Reply:
@p-m-j yup, no problem. Might not be today but will sort asap 👍🏻